features/events/syslog/src/main/resources/org/opennms/netmgt/syslogd/grok-patterns.txt

```diff
@@ -78,3 +81,15 @@
 
 # Netgear variant of RFC 3164 with day of week and no hostname or process info
 <%{INT:facilityPriority}> %{STRING:dayofweek}, %{INT:year}-%{INT:month}-%{INT:day} %{INT:hour}:%{INT:minute}:%{INT:second} - %{STRING:message}
+
+# Cisco syslog: https://www.cisco.com/c/en/us/td/docs/routers/access/wireless/software/guide/SysMsgLogging.html#wp1054470
+<%{INT:facilityPriority}>%{INT:parmSequenceNum}: %{NOSPACE:parmComponentId}:%{MONTH:month} %{INT:day} %{INT:hour}:%{INT:minute}:%{INT:second}.%{INT:secondFraction} : %{NOSPACE:processName}[%{INT:processId}]: %{STRING:message}
+<%{INT:facilityPriority}>: %{INT:year} %{MONTH:month} %{INT:day} %{INT:hour}:%{INT:minute}:%{INT:second} %{STRING:timezone}: %{STRING:message}
+<%{INT:facilityPriority}>%{INT:parmSequenceNum}: %{STRING:month} %{INT:day} %{INT:hour}:%{INT:minute}:%{INT:second}.%{INT:secondFraction}: %{STRING:message}
+<%{INT:facilityPriority}> %{INT:year} %{MONTH:month} %{INT:day} %{INT:hour}:%{INT:minute}:%{INT:second} %{STRING:timezone} *%{HOSTNAME:hostname}* %{STRING:message}
+<%{INT:facilityPriority}> %{INT:year} %{MONTH:month} %{INT:day} %{INT:hour}:%{INT:minute}:%{INT:second} %{STRING:timezone} %{STRING:message}
+
+# Note: This pattern '%{STRING:message}' should always come last as it will greedily match anything. If a pattern is
+# placed after this one it will never be matched since this pattern matches anything.
+# TODO: Update this to take any string beginning with '%' i.e. %{BEGINSWITH_%:message}
```

Comment:
It may not be a bad thing to leave this here without matching `%` first. This way the message won't be dropped, and given the new behavior we've introduced it will be associated with the current time, and the source host.